src/common/Input/Input.js

```diff
@@ -120,6 +142,28 @@ const Input = React.forwardRef(
             {inputIcon}
           </span>
         )}
+
+        {suggestionList?.length > 0 && inputIsFocused && (
+          <ul className="suggestion-list">
+            {suggestionList.map((item, index) => {
+              return (
+                <li
+                  className="suggestion-item"
+                  key={item + index}
```

Comment:
For string concatenation please prefer using string-template literals rather than the binary `+` operator.
```suggestion
                  key={`${item}${index}`}
```